### ISSUE TITLE
Feature: Add a feature to auto-upload the recorded videos to Google Drive

### ISSUE THREAD

**Issue by anonfaded** [anonfaded/FadCam#5]
This can be done after #4

**Comment by Lordlorio** [comment]
@anonfaded  Would it also be possible to add the upload of multiple videos simultaneously, but manually? Because every time we want to synchronize a video, we have to click on the three small dots for each video manually, which is not very practical.Thank you for your work.

**Comment by anonfaded** [comment]
> [@anonfaded](https://github.com/anonfaded) Would it also be possible to add the upload of multiple videos simultaneously, but manually? Because every time we want to synchronize a video, we have to click on the three small dots for each video manually, which is not very practical.Thank you for your work.

Yes there are improvements possible, at the moment i made a quick option so until the robust implementation is made, user can at least upload one by one. I will work on this soon once the openGL setup is done, it is almost complete:)
